lang/lang_en_de.txt

```diff
@@ -72,7 +72,7 @@
 
 #MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
 "Autoloading filament available only when filament sensor is turned on..."
-"Automatisches Laden Filament nur bei einge schaltetem Filament- sensor verfuegbar..."
+"Automatisches Laden Filament nur bei einge schaltetem Filamentsensor verfuegbar..."
```

Comment:
We had to shorten the text to fit on one screen in 3.8.1. Please test and report if it is already ok.